enterprise/hll/src/main/java/io/crate/operation/aggregation/HyperLogLogDistinctAggregation.java

```diff
@@ -88,7 +88,7 @@ private HyperLogLogDistinctAggregation(FunctionInfo info, DataType dataType) {
     public HllState newState(RamAccountingContext ramAccountingContext,
                              Version indexVersionCreated,
```

Comment:
I wonder if we should also remove the `indexVersionCreated`  from the interface. The HLL function was the only reason we added that. (and we were reluctant to do so)

Reply:
ah ok, I was not sure about that. I guess it makes sense to remove it then.

Reply:
here's the diff: https://gist.github.com/chaudum/d6754e3c74776c04ac76cf645787353f